src/editor/components/codemirror-fetch-mode.js

```diff
@@ -6,10 +6,13 @@ CodeMirror.defineSimpleMode("fetch", {
     { regex: /\/\/.*/, token: "comment", sol: true },
     { regex: /[\s]+\/\/.*/, token: "comment" },
     {
-      regex: /(blob: |text: |json: )(\w+)(\s*=\s*)(\S+)/,
+      regex: /(blob: |text: |json: |arrayBuffer: )(\w+)(\s*=\s*)(\S+)/,
       token: ["fetch-type", "fetch-variable-declaration", null, "fetch-path"]
     },
     { regex: /(js: |css: )(\S+)/, token: ["fetch-type", "fetch-path"] },
-    { regex: /text: |blob: |json: |js: |css: /, token: "fetch-type" }
+    {
+      regex: /text: |blob: |json: |js: |css |arrayBuffer: /,
```

Comment:
missing colon after css?

Reply:
thanks for the catch :) went ahead and verified the rest of them work as well.